homeassistant/components/kodi/media_player.py

```diff
@@ -641,13 +641,12 @@ def extra_state_attributes(self) -> dict[str, str | None]:
         if self.state == MediaPlayerState.OFF:
             return state_attr
 
-        hdr_type = (
-            self._item.get("streamdetails", {}).get("video", [{}])[0].get("hdrtype")
-        )
-        if hdr_type == "":
-            state_attr["dynamic_range"] = "sdr"
-        else:
-            state_attr["dynamic_range"] = hdr_type
+        state_attr["dynamic_range"] = "sdr"
+        video_details = self._item.get("streamdetails", {}).get("video", [{}])
+        if video_details:
+            hdr_type = video_details[0].get("hdrtype")
+            if hdr_type:
+                state_attr["dynamic_range"] = hdr_type
```

Comment:
Looks good, small suggestion for the syntax:

```suggestion

        state_attr["dynamic_range"] = "sdr"
        if (video_details := self._item.get("streamdetails", {}).get("video")) and (
            hdr_type := video_details[0].get("hdrtype")
        ):
            state_attr["dynamic_range"] = hdr_type
```